src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -242,12 +242,9 @@ object TxPoolConfig {
 trait MiningConfig {
```

Comment:
Minor detail: Both `activeTimeout` and `mineRounds` are not being used, should we maybe remove them?

Reply:
:+1: